news/2021-02-04-modding-mentorship-cycle-2-signups-now-open

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: Modding Mentorship Cycle 2: Signups Now Open
+date: 2021-02-04 10:00:00 +0000
+---
+
+Greetings, modders! With the new year starting, we want to announce the second run of Modding Mentorship! If you ever wanted to polish your modding abilities or assist newer members of modding communities in their way to achieve the needed modding skills, this is your time to shine!
+
+The Modding Mentorship is a place where new and promising modders of each gamemode can improve their modding skills under the supervision of pre-selected pool of modding experts serving as mentors.
+
+During the cycle, you will get to know the basics of modding as well as more in-depth analysis from the mentors. You will also have an opportunity to participate in a variety of events (like modding puzzles).
+
+If you believe you've got what it takes to teach a new generation of modders, don't hesitate, and try yourself as a mentor! [The information thread](https://osu.ppy.sh/community/forums/topics/1218245) will guide you on how to apply right away! If you are just taking your first steps in the modding community, though, consider applying as a mentee!
+
+You may also want to join us at [the Modding Mentorship discord](https://discord.gg/TyvrMvR) to participate in the public events and keep up with the announcements.
+
+---
+
+For this cycle, the application deadlines will go as following:
+- Mentor signups will be open until **18th February, 2021**
+- Mentee signups will open on **18th February, 2021** and close on **25th February**
```

Comment:
```suggestion
For this cycle, the application deadlines will go as follows:
- Mentor signups will be open until **February 18, 2021**
- Mentee signups will open on **February 18, 2021** and close on **February 25**
```
these are the correct date & time formatting according to the [Article Styling Criteria](https://osu.ppy.sh/wiki/en/Article_styling_criteria/Writing#date-and-time)